tests/unit/utils/test_jinja.py

```diff
@@ -612,6 +612,22 @@ def test_render_with_undefined_variable_unicode(self):
             dict(opts=self.local_opts, saltenv="test", salt=self.local_salt),
         )
 
+    def test_relative_include(self):
```

Comment:
Is this test really make sense? I see it's not failing even without thix fix applied.

Reply:
IMHO makes sense. You can reproduce the fail when installing jinja 3.0.1 that is in TW. This test fails with this Jinja, and succeed with 2.11.

Reply:
Oh, sorry. You are right. Missed that it's dependant on Jinja2 vertion used.